android/gradle.properties

```diff
@@ -1,5 +1,3 @@
 org.gradle.jvmargs=-Xmx1536M
-android.enableR8=true
 android.useAndroidX=true
 android.enableJetifier=true
-android.bundle.enableUncompressedNativeLibs=false
```

Comment:
This option is needed for targeting Android 6.0, don't remove it.

https://docs.flutter.dev/deployment/android#building-the-app-for-release

Reply:
I reverted it as you requested